assets/cubyz/biomes/cave/void_roots.zig.zon

```diff
@@ -1,15 +1,22 @@
 .{
 	.isCave = true,
 	.maxHeight = -50000,
-
-	.fogDensity = 2,
+	.fogDensity = 10,
+	.fogColor = 0x090511,
 	.chance = 100,
 	.caveRadiusFactor = -1,
 	.caves = 0.5,
 
 	.music = "cubyz:heart-of-the-beast",
 
 	.structures = .{
+		.{
+			.id = "cubyz:simple_vegetation",
+			.chance = 0.001,
+			.block = "cubyz:torch",
+			.height = 1,
+			.height_variation = 0,
+		},
```

Comment:
Wow, what a nice surprise!
You really got me there for a moment, wondering why I had gone to the root in this random world from your fork.

But honestly I think we should make it a bit rarer still.

Also another idea: You could use a 2-block tall tree with a workbench for the logs, a torch for the top and no leaves to create a crafting table with a torch on top.

Reply:
I don't know, maybe 10× rarer?